sycl/include/sycl/types.hpp

```diff
@@ -674,13 +690,7 @@ template <typename Type, int NumElements> class vec {
 
   vec() = default;
 
-  // TODO Remove this difference between host and device side after
-  // when root cause of API incompatibility will be fixed
```

Comment:
Is this referring to an issue this patch fixes?

Reply:
Yes.